Doctrine/Phpcr/SimpleUrlInformationGuesser.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2015 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\SeoBundle\Doctrine\Phpcr;
+
+use Symfony\Cmf\Bundle\SeoBundle\AlternateLocaleProviderInterface;
+use Symfony\Cmf\Bundle\SeoBundle\Model\UrlInformation;
+use Symfony\Cmf\Bundle\SeoBundle\SeoPresentation;
+use Symfony\Cmf\Bundle\SeoBundle\Sitemap\UrlInformationGuesserInterface;
+use Symfony\Component\Routing\RouterInterface;
+
+/**
+ * A base guesser to extract url information from a document persisted with the phpcr-odm.
+ *
+ * @author Maximilian Berghoff <Maximilian.Berghoff@gmx.de>
+ */
+class SimpleUrlInformationGuesser implements UrlInformationGuesserInterface
+{
+    /**
+     * @var AlternateLocaleProviderInterface
+     */
+    protected $alternateLocaleProvider;
+
+    /**
+     * @var RouterInterface
+     */
+    private $router;
+
+    /**
+     * @var string
+     */
+    private $defaultChangeFrequency;
+
+    /**
+     * @var SeoPresentation
+     */
+    private $seoPresentation;
+
+    /**
+     * @param RouterInterface $router
+     * @param SeoPresentation $seoPresentation
+     * @param $defaultChangeFrequency
+     */
+    public function __construct(
+        RouterInterface $router,
+        SeoPresentation $seoPresentation,
+        $defaultChangeFrequency
+    ) {
+        $this->router = $router;
+        $this->seoPresentation = $seoPresentation;
+        $this->defaultChangeFrequency = $defaultChangeFrequency;
+    }
+
+    /**
+     * @param AlternateLocaleProviderInterface $alternateLocaleProvider
+     */
+    public function setAlternateLocaleProvider(AlternateLocaleProviderInterface $alternateLocaleProvider)
+    {
+        $this->alternateLocaleProvider = $alternateLocaleProvider;
+    }
+
+    /**
+     * {@inheritDocs}
+     */
+    public function guessValues(UrlInformation $urlInformation, $object, $sitemap = 'default')
+    {
+        $urlInformation->setLocation($this->router->generate($object, array(), true));
```

Comment:
we should add some checks first, to see if the value isn't already guessed by another guesser and if this guesser should overrule that value.

Actually, I think we should use a real guess implementation with a confidence. E.g:
````php
$values = array();

$values[] = new LocationGuess(
    $this->router->generate($object, array(), true),
    Guess::VERY_HIGH_CONFIDENCE
);

$values[] = new ChangeFrequencyGuess(
    $this->defaultChangeFrequency,
    Guess::LOW_CONFIDENCE
);

// ...
````

The value with the highest confidence is then set in the UrlInformation instance. Just like https://github.com/symfony/Form/blob/master/Guess/Guess.php